pkgs/development/compilers/miranda/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, gcc }:
+
+stdenv.mkDerivation rec {
+  version = "2.066";
+  name = "${version}";
+
+  src = builtins.fetchTarball {
+    url = "https://www.cs.kent.ac.uk/people/staff/dat/miranda/src/mira-2066-src.tgz";
+    sha256 = "0d16ibf3c3j93bx781bz3ga8q15acmb9080haixxjnsqpx6cakr8";
+  };
+
+  enableParallelBuilding = true;
+
+  configurePhase = ''
+    make cleanup
+    patchShebangs quotehostinfo
+  '';
```

Comment:
Why is that? And also, read: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/explicit-phases.md